lib/tandem/tandemSimulator.js

```diff
@@ -47,6 +47,7 @@ exports.make = function(config){
 
   var currBasal = null;
   var currTimestamp = null;
+  var currStatus = null;
```

Comment:
Done (for now) using suspend/resume basals. Will be changed to use suspend/resume events when we're reading them in next sprint.